tasK_1.py

```diff
@@ -0,0 +1,27 @@
+"""
+Задача 1: Напишите программу, которая получает целое число и возвращает его шестнадцатеричное
+строковое представление. Функцию hex используйте для проверки своего результата.
+"""
+
+SYSTEM: int = 16
+result: str = "" 
+
+while True:
+    number = input("Введите любое число: ")
+    if not number.isdigit():
+        print("Попробуйте еще...")
+        continue
+    break
+
+number = int(number)
+if number == 0: result = "0"
```

Comment:
Согласно PEP8, такая запись, хоть и допустима, но не принята
```suggestion
if number == 0:
    result = "0"
```